pyproject.toml

```diff
@@ -124,36 +124,22 @@ feff_plot_dos = "pymatgen.cli.feff_plot_dos:main"
 get_environment = "pymatgen.cli.get_environment:main"
 
 [tool.setuptools]
-include-package-data = false
+include-package-data = true
+
+[tool.setuptools.exclude-package-data]
```

Comment:
WIP: this section is not working for some reason, those files are still included in the wheel